src/tasks/batch_integration/api/file_integrated_graph.yaml

```diff
@@ -11,8 +11,16 @@ info:
         name: connectivities
         description: Neighbors connectivities matrix.
         required: true
+      - type: double
+        name: distances
+        description: Neighbors connectivities matrix.
+        required: true
     uns:
       - type: string
         name: method_id
         description: "A unique identifier for the method"
         required: true
+      - type: object
+        name: neighbors
+        description: Supplementary K nearest neighbors data.
+        required: true
```

Comment:
The graph metrics didn't want to work if the graph methods don't output an `.uns["neighbors"]` object. Is this intended? Because I think it essentially means that the graph method output always needs to be produced by scanpy's neighbors function.